packages/ListBox/src/hooks/hooks.js

```diff
@@ -103,10 +105,10 @@ export function useOnScrolled() {
           offsetTop = 0;
         }
 
-        state.refListBox.current.scrollTo(0, offsetTop - 10);
+        if (state.refListBox.current) state.refListBox.current.scrollTo(0, offsetTop - 10);
```

Comment:
Btw I thought `scrollTo` is not supported in IE?

Reply:
correct [see](https://github.com/acl-services/paprika/pull/205#issuecomment-535198493), I added a polyfill for Edge, should work on IE11 as well but currently storybook is not working anymore on it.